$(var namespace)/map_merge:
  ros__parameters:
    # Map Topic discovery
    discovery_rate: 0.5
    map_topic: map
    map_namespace: robot

    # Map matching
    matching_rate: 1.0 #0.2
    matching_confidence: 0.7 #0.3

    # Map merging
    merging_rate: 2.0
    publish_merged_map: true
    publish_tf: true
    merged_map_topic: world_map
    world_frame: world

    # Setting own map to world zero position
    # ssugv1/init_pose_x: 0.0
    # ssugv1/init_pose_y: 0.0
    # ssugv1/init_pose_z: 0.0
    # ssugv1/init_pose_yaw: 0.0

    # ssugv2/init_pose_x: 0.0
    # ssugv2/init_pose_y: 0.0
    # ssugv2/init_pose_z: 0.0
    # ssugv2/init_pose_yaw: 0.0

    # ssugv3/init_pose_x: 0.0
    # ssugv3/init_pose_y: 0.0
    # ssugv3/init_pose_z: 0.0
    # ssugv3/init_pose_yaw: 0.0


    $(var namespace)/init_pose_x: 0.0
    $(var namespace)/init_pose_y: 0.0
    $(var namespace)/init_pose_z: 0.0
    $(var namespace)/init_pose_yaw: 0.0

    robot2/init_pose_x: 0.0
    robot2/init_pose_y: 0.0
    robot2/init_pose_z: 0.0
    robot2/init_pose_yaw: 0.0

    # ssugv3/init_pose_x: -4.37
    # ssugv3/init_pose_y: -0.70
    # ssugv3/init_pose_z: 0.0
    # ssugv3/init_pose_yaw: 0.0
